### DISCUSSION TITLE
How do you handle Robot Protection?

### DISCUSSION THREAD

**Discussion by mirisbowring** [karakeep-app/karakeep#484] [Q&A] [answered]
I started using this awesome app by collecting some links (i am currently serching for a specific product). Some links are e.g. from Etsy but hoarder fails to get an image because etsy wants to know whether hoarder is a robot :D

![image](https://github.com/user-attachments/assets/494cc612-8f3c-4892-a63b-2af67fb81cbf)

**Comment by kamtschatka** [comment, marked as answer]
This is a cat and mouse game, which can not be won. The whole point is to keep robots out, so if you try to avoid it, it will just mean that they will improve as well. 
There was an idea to allow capturing the data from the browser extension instead, so even if you would be detected as a robot, you can simply finish the CAPTCHA and still capture data.

**Reply by mirisbowring** [reply]
Ah, expected that :) A "get image from browser" (via extension) sounds great though :)

**Reply by grmrgecko** [reply]
Keep me up to date on capturing page from the browser, because I have found some links to dislike hoarder such as https://www.digikey.com/en/products/detail/linear-technology/LT3580IMS8E-PBF/1730237

**Reply by matttk** [reply]
Can't we at least upload a custom image?

**Reply by francorbacho** [reply]
Second uploading custom image.
